pkg/cli/cli.go

```diff
@@ -145,7 +146,10 @@ func (c *CLI) buildCmd() error {
 	c.cmd = c.newRootCmd()
 
 	// Get project version and plugin keys.
-	if err := c.getInfo(); err != nil {
+	// Ignore 3-alpha error
+	switch err := c.getInfo(); {
+	case err == nil:
+	case !errors.Is(err, config.UnsupportedVersionError{Version: config.Version{Number: 3, Stage: stage.Alpha}}):
```

Comment:
What do you think of something a little bit more generic like this? (You need to declare the variable `unsupportedVersionErr` of type `config.UnsupportedVersionError` above the `switch`)
```suggestion
	case errors.As(err, &unsupportedVersionErr):
		if !unsupportedVersionErr.Version.IsStable() {
			// Check if the corresponding stable version exists, set c.projectVersion and break
		}
		fallthrough
	default:
```

Reply:
How about returning any `UnsupportedVersionError` to the caller so they can handle it appropriately?

Also this doesn't solve the general case of a command that can be run outside of a project.

Reply:
That was already solved if I understood you correctly. The problem is faced when a PROJECT with a unknown version is found (`3-alpha`) not when no PROJECT file is found. And returning an error makes the command fail, we need to skip for 3-alpha (or in the generic solution I propose here, for any unregistered unstable version whose stable version is registered).

Reply:
@Adirio @estroz correct, a command that has no PROJECT file already works. That is checked explicitly.